CMakeLists.txt

```diff
@@ -100,3 +100,41 @@ else()
         endif()
     endif()
 endif()
+
+#Added Qt6 support via QT_ANDROID_EXTRA_LIBS
```

Comment:
I suggest this commit also update the readme to talk about how to include this when MANUAL_FINALIZATION is used with qt_add_executable() (https://doc.qt.io/qt-6/qt-add-executable.html#synopsis), because this might not take effect if MANUAL_FINALIZATION is set and the include is done after calling qt_finalize_target()

Reply:
It should be like this, according to Qt example in the CMake file.
I set qt_finalize_target at the end of file after qt_finalize_executable and everything works fine.

```
qt_add_executable(${PROJECT_NAME}
    MANUAL_FINALIZATION
    ${PROJECT_SOURCES}
)

target_link_libraries(${PROJECT_NAME}
  PRIVATE Qt${QT_VERSION_MAJOR}::Core Qt${QT_VERSION_MAJOR}::Quick)

if(ANDROID)
    include(${ANDROID_SDK_ROOT}/android_openssl/CMakeLists.txt)
endif()

install(TARGETS ${PROJECT_NAME}
    BUNDLE DESTINATION .
    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})

if(QT_VERSION_MAJOR GREATER_EQUAL 6)
    qt_import_qml_plugins(${PROJECT_NAME})
    qt_finalize_executable(${PROJECT_NAME})
    qt_finalize_target(${PROJECT_NAME})
endif()

```

Reply:
exactly, that would work, but I meant if someone add the include() call after calling qt_finalize_target(), the openssl libs won't be included because QT_ANDROID_EXTRA_LIBS will not be evaluated again after the finalize call, and I thought documenting that might be useful

Reply:
I completely agree with your suggestion.
I'll add an explanation about this to the code and I hope that explanations related to this section will be added in the main documentation.